### PR TITLE
Add dask-geomodeling recipe

### PR DESCRIPTION
**Pull request by caspervdw**
This is an initial attempt of creating the dask-geomodeling conda distribution. @reinout

### CODE REVIEW

**Comment by caspervdw** [comment]
That seems to be done automatically by the cf-autotick-bot

Like here: https://github.com/conda-forge/trackpy-feedstock/pull/10

There is a small difference in the repos where the sha256 is stored, but as I just took the default example, I guess this will be fine.

**Comment by caspervdw** [comment]
@conda-forge/help-python  This can be merged

File: recipes/dask-geomodeling/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: da7f1450ba8ac87f61006b49aba8f341d742ca940fdb4d36ed4b783d694bfc7f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
```
Comment (marcelotrevisani) [inline recipes/dask-geomodeling/meta.yaml]: I think this recipe can be ``noarch``
```suggestion
  noarch: python
```

You can restrict the python version on the requirements

File: recipes/dask-geomodeling/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+requirements:
+  host:
+    - python
```
Comment (marcelotrevisani) [inline recipes/dask-geomodeling/meta.yaml]:
```suggestion
    - python >=3.5
```

File: recipes/dask-geomodeling/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    - pip
+  run:
+    - python
```
Comment (marcelotrevisani) [inline recipes/dask-geomodeling/meta.yaml]:
```suggestion
    - python >=3.5
```

File: recipes/dask-geomodeling/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
```
Comment (marcelotrevisani) [inline recipes/dask-geomodeling/meta.yaml]: Could you please remove all the unnecessary comments?

**Comment by caspervdw** [comment]
Thanks for you review @marcelotrevisani 

I encountered some Python 3.5 compatibility issues, so this should still wait for https://github.com/nens/dask-geomodeling/pull/4 to be merged and released.

**Comment by caspervdw** [comment]
@marcelotrevisani  This is done and ready to be merged

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**